indy-utils/build.gradle

```diff
@@ -1,12 +1,11 @@
 dependencies {
-    compile group: 'org.hyperledger', name: 'indy', version: "$indy_version"
+    compile 'com.github.seniorjoinu:indy-sdk:v1.7.0-indy-android-compat-5'
```

Comment:
Need to migrate to original indy-sdk

Reply:
It's another research topic (there should be a lot of gradle magic to make it work), so I suggest to move this to the backlog.

Reply:
let's do that